tally_ho/urls.py

```diff
@@ -24,24 +23,26 @@
         },
         name='login'),
     url(r'^password_change/$',
-        'django.contrib.auth.views.password_change',
+        auth_views.PasswordChangeView.as_view(),
         {
             'password_change_form': PasswordChangeForm,
             'post_change_redirect': '/'},
         name='password_change'),
     url(r'^password_change/done/$',
-        'django.contrib.auth.views.password_change_done',
+        auth_views.PasswordChangeDoneView.as_view(),
         name='password_change_done'),
-    url(r'^logout/$', auth_views.logout, {'next_page': '/'}, name='logout'),
-)
+    url(r'^logout/$',
+        auth_views.LogoutView.as_view(),
+        {'next_page': '/'},
+        name='logout'),
+]
 
 handler403 = 'tally_ho.apps.tally.views.home.permission_denied'
 handler404 = 'tally_ho.apps.tally.views.home.not_found'
 handler400 = 'tally_ho.apps.tally.views.home.bad_request'
 handler500 = 'tally_ho.apps.tally.views.home.server_error'
 
-urlpatterns = patterns(
-    '',
+urlpatterns = [
     url(r'^$', home.HomeView.as_view(), name='home'),
```

Comment:
I think we need to use the new `path` or `re_path`: https://docs.djangoproject.com/en/2.1/ref/urls/#path